app/Services/PayPalService.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Services;
+use Illuminate\Http\Request;
+use Laravel\Cashier\Subscription;
+use Carbon\Carbon;
+use Srmklive\PayPal\Services\PayPal as PayPalClient;
+
+class PayPalService
+{   
+    /**
+    * @param Request $request
+    * @return mixed
+    */
+   public function process(Request $request)
```

Comment:
If I change in the HTML the pledge to "banana" before submitting the form, it crashes and will send me an email on prod, which I'd like to avoid by having some validation :)